src/plugins/robots/drone/control_interface/ci_drone_flight_system_sensor.cpp

```diff
@@ -34,8 +37,11 @@ namespace argos {
       lua_getfield(pt_lua_state, -1, "flight_system");
       CLuaUtility::AddToTable(pt_lua_state, "position", GetPosition());
       CLuaUtility::AddToTable(pt_lua_state, "orientation", GetOrientation());
+      CLuaUtility::AddToTable(pt_lua_state, "target_orientation", GetTargetOrientation());
       CLuaUtility::AddToTable(pt_lua_state, "velocity", GetVelocity());
       CLuaUtility::AddToTable(pt_lua_state, "angular_velocity", GetAngularVelocity());
+      CLuaUtility::AddToTable(pt_lua_state, "target_position", GetTargetPosition());
+      CLuaUtility::AddToTable(pt_lua_state, "lidar_height_measurment", GetDistanceSensorData());
```

Comment:
How about in Lua, we just have `height` and in C++ we have `GetHeight()` and `m_fHeight`? This is consistent with variables like `target_orientation`, `GetTargetOrientation()` and `m_fTargetOrientation`. Putting "Lidar" or "distance sensor" in the variable name isn't really important to this code. I think this code should treat the Pixhawk like a black box.

Reply:
sounds good!